### DISCUSSION TITLE
stars as solid disk?

### DISCUSSION THREAD

**Discussion by rockscientist01** [Stellarium/stellarium#1797] [Q&A] [answered]
Hello,
is there a setting that stars are shown as solid disks not fading to the edges?
Thanks

**Comment by gzotti** [comment, marked as answer]
No. There are two options, a soft-edged disk and a spiky star. Solid disks can be found with other programs.

**Reply by rockscientist01** [reply]
I assume I'm not the first person to ask about this feature? MAy I ask why this is not an option? It would make printed skycharts much more readable under a dark sky.

**Reply by gzotti** [reply]
Stellarium's strongest point is not printed sky charts. Various astro fora discuss optimized printing mostly with other, also free programs. Stellarium is about nicer images with many colors between black and white. Presumably most users say "good enough" to expendable printouts. I hope you create inverted screenshots, black stars on white ground?

**Reply by rockscientist01** [reply]
OK understood. Yes, I inverted colors for printouts. I love stellarium, just being curious. Thanks for your answer and clarification. 👍

**Reply by axd1967** [reply]
see also #1662
